.github/workflows/smoke-test-imperative-compare.yml

```diff
@@ -0,0 +1,124 @@
+          - imperative-ngx-jwt-maven-psql-nocache
+          - imperative-react-jwt-gradle-mysql-caffeine
+          - imperative-ngx-jwt-maven-mysql-redis
+          - imperative-sql-jwt-maven-no-db
+          - imperative-ngx-oauth2-maven-mysql-caffeine
+        include:
+          - app: imperative-ngx-jwt-maven-mysql-caffeine
+          - app: imperative-ngx-jwt-maven-mongodb-caffeine
+          - app: imperative-ngx-jwt-maven-psql-nocache
+          - app: imperative-react-jwt-gradle-mysql-caffeine
+          - app: imperative-ngx-jwt-maven-mysql-redis
+          - app: imperative-sql-jwt-maven-no-db
+          - app: imperative-ngx-oauth2-maven-mysql-caffeine
+        exclude:
+          - app: imperative-ngx-jwt-gradle-mongodb-redis
+    steps:
+      - name: 'SETUP: Checkout quarkus blueprint'
+        uses: actions/checkout@v2
+        with:
+          path: generator-jhipster-quarkus
+          fetch-depth: 5
+      - name: git history
+        run: git --no-pager log -n 10 --graph --pretty='%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue) <%an>%Creset' --abbrev-commit
+        working-directory: ./generator-jhipster-quarkus
+      - name: 'SETUP: Checkout generator-jhipster'
+        uses: actions/checkout@v2
+        with:
+          repository: 'jhipster/generator-jhipster'
+          path: generator-jhipster
+      - name: 'SETUP: environment'
+        id: setup
+        uses: ./generator-jhipster/.github/actions/setup
+        with:
+          jdl-sample: ${{ github.workspace }}/generator-jhipster-quarkus/test-integration/samples/${{ matrix.app }}.jdl
+          generator-jhipster-branch: local
+          jhipster-bom-branch: ignore
+          package-with-executable: generator-jhipster-quarkus
+          executable: jhipster-quarkus
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ steps.setup.outputs.node-version }}
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: ${{ steps.setup.outputs.java-version }}
+      - name: 'Generate: Project'
+        uses: ./generator-jhipster/.github/actions/generate
+      #----------------------------------------------------------------------
+      # Generate project
+      #----------------------------------------------------------------------
+      #----------------------------------------------------------------------
+      # Detect changes against base commit
+      #----------------------------------------------------------------------
+      - name: 'MERGE: generate base'
+        continue-on-error: true
+        id: base-app
+        if: github.event.pull_request
+        uses: ./generator-jhipster/.github/actions/compare-base
+      - name: 'MERGE: compare changes'
+        continue-on-error: true
+        id: compare
+        if: steps.base-app.outcome == 'success'
+        uses: ./generator-jhipster/.github/actions/compare
+      - name: 'Backend diff with updated prettier'
```

Comment:
Why do we need this step?

Reply:
v6 -> v7 was complicated and all files changed due to prettier changes.
It was temporary added for this purpose. Can be dropped.